### PR TITLE
OSDOCS-9941: Correction to the ROSA with HCP cluster creation process

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-9941](https://issues.redhat.com//browse/OSDOCS-9941): Correction to the ROSA with HCP cluster creation process

Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-9941

Link to docs preview:
ROSA: https://73024--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly#rosa-sts-creating-account-wide-sts-roles-and-policies_rosa-hcp-sts-creating-a-cluster-quickly
1. Account Roles
    image

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc [hunk 1]
```diff
@@ -30,7 +30,21 @@ $ rosa create account-roles --hosted-cp --mode auto --yes
...
+[source,terminal]
+----
+$ echo $ACCOUNT_ROLES_PREFIX
+----
++
+.Sample output
++
+[source,terminal]
+----
+$ ManagedOpenShift
```
Comment (EricPonvelle) [inline modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc:47]: I don't think you need the `$` for return values. I have also seen people use "source,text" for the example box. Just an idea.

File: modules/rosa-sts-byo-oidc.adoc [hunk 2]
```diff
@@ -78,7 +78,7 @@ $ echo $OIDC_ID
 +
 [source,terminal]
 ----
-$ 30f5dqmk
+$ 13cdr6b
```
Comment (EricPonvelle) [inline modules/rosa-sts-byo-oidc.adoc:81]: These seem fine, though I wonder if you should have an example built out where it shows "export OIDC_ID=13cdr6b" and then you have this response? 

Same feedback about the "source,text" and `$` here.

**Review by EricPonvelle — COMMENTED** [review]
Small nits for your consideration

**Comment by jneczypor** [comment]
Hi @yuwang-RH - Can I please have you review on this PR? Thanks!

**Comment by yuwang-RH** [comment]
lgtm

**Review by arendej — APPROVED** [review]
lgtm

File: modules/rosa-sts-byo-oidc.adoc [hunk 3]
```diff
@@ -60,25 +60,26 @@ I: Created OIDC provider with ARN 'arn:aws:iam::4540112244:oidc-provider/dvbwgdz
...
 +
 [source,terminal]
 ----
 $ echo $OIDC_ID
 ----
 +
-.Sample output
+.Example output
 +
 [source,terminal]
```
Comment (EricPonvelle) [inline modules/rosa-sts-byo-oidc.adoc:80]: Can you change this source to `text`?

Comment (jneczypor) [reply modules/rosa-sts-byo-oidc.adoc:80]: I am going to leave it as `[source, terminal]` per the [OpenShift contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets).
Screenshot 2024-03-21 at 3 35 17 PM

**Comment by bchandra-ocp** [comment]
It is not related to this JIRA/changes but the content on this page seems repetitive. Why do we repeat the table comparing Classic and HCP or network diagrams when they are already in the Understanding ROSA page(s)?

**Comment by jneczypor** [comment]
Hi @bchandra-ocp - Good catch! This PR is separate from OSDOCS-9589, which means any changes made in OSDOCS-9589 are not reflected here yet. I moved the table in that PR, but it is still in the old location for this PR. Once that OSDOCS-9589 merges, the content will be a lot less repetitive. Thanks.

**Comment by bchandra-ocp** [comment]
Sounds good. Also, will that remove the diagrams about Classic ROSA?

**Comment by jneczypor** [comment]
Yes, it should move all of that content.

**Pull request merged**